### ISSUE TITLE
Errors in es_ES and ca_ES translations in OJS 3.2.x

### ISSUE THREAD

**Issue by marcbria** [pkp/pkp-lib#6912]
**Describe the bug**
Weblate translations are done against last stable (today 3.3.x) and former versions are frozen, but sometimes an old versions requires a translation fix because the text is relevant enough to make the tool usage confusing.
Is not feasible create a new release just for a translation change, but probably is not a bad idea introduce it them when a security patch is released and a new package is created?

A translation error could appear in a lot of old versions, but we are suggesting only fix it on stable releases.

As an example (and also the detonator of this petition) is the error found at:
https://github.com/pkp/pkp-lib/blob/0fc2f22420eb7f6c5e581e29debc6862c5a98810/locale/es_ES/submission.po#L1527

Where the English string:
```
"Submissions in several languages are accepted. Choose the primary language of the submission from the pulldown above."
```

Was wrongly translated as:
```
"Submissions in several languages are accepted. Choose the primary language of the submission from the pulldown BELOW."
```

So our testing show that users are confused at this point not been to find the dropdown easily.

A PR against the last 3.2.x stable will be done, just in case the translation fix is accepted to be packaged.

**Comment by asmecher** [comment]
@marcbria, I've merged both the above PRs; I know you were talking i part about a general problem, but having merged those I'll close this issue; if you come across another translation needing work let's fix it separately. I've also cherry-picked this change forward to `stable-3_3_0` so it's not lost there.

Have you made these changes in Weblate for the `main` branch?

**Comment by marcbria** [comment]
Yes. Jordi fixed this in weblate too.
So far, for us, weblate is working great.
Thanks for the merge. It will be on 3.2.1-5, isn't it?

**Comment by asmecher** [comment]
@marcbria, we probably won't get around to releasing a 3.2.1-5, but it'll definitely be included in 3.3.0-5 (probably today).

The translation tools currently apply to `main` and I'm not back-porting translation changes lately. Once 3.4.0 comes out we'll likely leave the translation tools active against `stable-3_4_0` so the follow-up changes after the release will go into both 3.4.x and and `main` (until we launch active development of 3.5.x).

Using `git cherry-pick` is probably the fastest way of applying changes to different branches if you have commit privileges; I'm not sure if there's something quick/easy for working with pull requests. Unfortunately Weblate doesn't have support for multiple branches; you're forced to choose the most appropriate one and then cherry-pick/adapt changes to other branches using something outside Weblate. (This is no slight against the excellent Weblate toolset -- just something they don't support at the moment.)
